pkg/cloud/scope/elb.go

```diff
@@ -49,4 +49,6 @@ type ELBScope interface {
 
 	// ControlPlaneEndpoint returns AWSCluster control plane endpoint
 	ControlPlaneEndpoint() clusterv1.APIEndpoint
+
+	SecondaryControlPlaneLoadBalancer() *infrav1.AWSLoadBalancerSpec
```

Comment:
If we add a function in this interface that returns all load balancers, could we do away with the specific functions?